apps/frontend/src/network/__tests__/TeamList.test.tsx

```diff
@@ -67,7 +54,19 @@ it('renders a loading indicator', async () => {
 });
 
 it('renders a list of teams information', async () => {
+  nock(API_BASE_URL, {
+    reqheaders: { authorization: 'Bearer token' },
+  })
+    .get('/teams')
+    .reply(
+      200,
+      createTeamListResponse(1).items.map((item) => ({
+        ...item,
+        displayName: 'Unknown',
+        projectTitle: 'Project Title Unknown',
+      })),
+    );
   const { container } = await renderTeamList();
-  expect(container.textContent).toContain('Unknown Team');
-  expect(container.textContent).toContain('Unknown Project Title');
+  expect(container.textContent).toContain('Unknown');
```

Comment:
This will already be fulfilled by the `Project Title Unknown` :D make it `Name Unknown` or so ^^